app/packages/operators/src/built-in-operators.ts

```diff
@@ -146,6 +146,8 @@ class OpenPanel extends Operator {
     if (node.hasChildren()) {
       return this.findFirstPanelContainer(node.firstChild());
     }
+
+    return null;
```

Comment:
out of the scope of PR, but can we also update the return type on line 141: `SpaceNode | null`